


try {
  console.log('hello')
} catch (error) {
  
}